src/main/resources/assets/twilightforest/patchouli_books/guide/en_us/entries/treasures/fiery_blood_tears.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Fiery Blood and Tears",
+  "icon": "twilightforest:fiery_blood",
+  "category": "treasures",
+  "advancement": "twilightforest:alt/treasures/fiery_blood_tears",
+  "sortnum": 5120,
+  "pages": [
+    {
+      "type": "twilightforest:gallery",
+      "title": "Fiery Blood and Tears",
+      "item_list": "twilightforest:fiery_blood|twilightforest:fiery_tears",
+      "text": "Small vials containing a hot liquid. Though containing different liquids, the properties have the same, burning properties, applicable for making equipment burn enemies. It seems to come from two powerful monsters."
```

Comment:
`Though containing different liquids, the properties have the same, burning properties, applicable for making equipment burn enemies.`

Descriptor is vague and uses recursive descriptor targets and overzealous commas. Also it's more for burning enemies, so maybe have more of radiating extreme heat or something similar.

Also perhaps `Small vials containing a hot liquid` -> `Small vials containing a hot bodily fluid`?

`from two powerful monsters` -> just `from powerful monsters`

Reply:
I think "from some powerful monsters" would be better because then it's clear that not *very* powerful monster drops it. 

Strongly object to "bodily fluid" because most people are going to assume you're talking about semen.

Reply:
LOL Fair enough on the `bodily fluid` bit

Reply:
"The very essence contained in these small vials of liquid is so imbued with heat that it can be used for the process of crafting metal likewise imbued with heat. It can be obtained from multiple powerful monsters."